syncopy/nwanalysis/connectivity_analysis.py

```diff
@@ -378,33 +380,35 @@ def connectivityanalysis(data, method="coh", keeptrials=False, output="abs",
     # ----------------------------------------------------------------------------------
     # Sanitize output and call the chosen ComputationalRoutine on the averaged ST output
     # ----------------------------------------------------------------------------------
-
-    out = CrossSpectralData(dimord=st_dimord)
-
-    # now take the trial average from the single trial CR as input
-    av_compRoutine.initialize(st_out, out._stackingDim, chan_per_worker=None)
-    av_compRoutine.pre_check()   # make sure we got a trial_average
-    av_compRoutine.compute(st_out, out, parallel=kwargs.get("parallel"),
-                           log_dict=log_dict)
-
-    # attach potential older cfg's from the input
-    # to support chained frontend calls..
-    out.cfg.update(data.cfg)
-    # attach frontend parameters for replay
-    out.cfg.update({'connectivityanalysis': new_cfg})
-    return out
+    if method == 'csd':
+        new_cfg.update({'output': st_out.data.dtype.name})
```

Comment:
What we want to save in the `cfg` are parameter values which can be re-used for the same frontend call, what you are saving here won't work, `output` has to be one of `connectivity_outputs` and already gets attached correctly here:
https://github.com/esi-neuroscience/syncopy/blob/83455e70a94369c900e1688033c0766ac6565751/syncopy/nwanalysis/connectivity_analysis.py#L219
So this line can be removed.